karma.conf.js

```diff
@@ -13,8 +13,7 @@ module.exports = function (config) {
     files: [
       'Layout.js',
       'Layout-test-utils.js',
-      '__tests__/Layout-test.js',
-      '__tests__/Layout-consts-test.js'
+      '__tests__/*.js'
```

Comment:
Can you not run random layout tests? They are mostly used for development purposes. The real tests that we support are in Layout-test.js